x-pack/plugins/security_solution/public/detection_engine/rule_details_ui/pages/rule_details/execution_log_table/execution_log_columns.tsx

```diff
@@ -139,6 +156,34 @@ export const getMessageColumn = (width: string) => ({
   width,
 });
 
+export const getSourceEventTimeRangeColumns = () => [
+  {
+    name: (
+      <TableHeaderTooltipCell
+        title={i18n.COLUMN_SOURCE_EVENT_TIME_RANGE}
+        tooltipContent={i18n.COLUMN_SOURCE_EVENT_TIME_RANGE_TOOLTIP}
+      />
+    ),
+    field: 'backfill',
+    render: (backfill: { to: string; from: string }) => {
```

Comment:
Seems like `FoemattedDate` handles this, but anyway according to schema defined above, both `to` and `from` are optional and can be undefined.

Reply:
I upated schema, I keep backfill optional, but if it's present, then from and to also present